### ISSUE TITLE
Does not show up in settings extensions

### ISSUE THREAD

**Issue by AddictArts** [flybrotherlixiang/FBXCodeExtension#1]
I followed all the steps including setting the targets to me for signing. No luck though. Any ideas?

**Comment by Denages** [comment]
Exactly the same issue. Xcode 8.1

**Comment by flybrotherlixiang** [comment]
Sorry I didn't notice this issue until now.  

After building the extension target, did the new editor command showed up in the menu of XCode which has the gray icon ? @AddictArts @Denages

**Comment by AddictArts** [comment]
@flybrotherlixiang  No it does not. I am choosing automatically manage code signing, My Name (Personal Team).

**Comment by AddictArts** [comment]
I think you have to have a valid Apple developer ID. My account is somehow tied to my employers enterprise account. That dev license is expired.

**Comment by flybrotherlixiang** [comment]
I guess that's why it won't show up... I've tried many times at first and nothing happened before I signed it with a valid certificate

**Comment by AddictArts** [comment]
It worked finally. Could have been both or one of these things. First I went to the keychain and deleted expired certs. This left a cert for myself that was Mac Developer and still valid. Not completely sure when I got that one. Then I did a "release" build and viola it worked. So, was it the release build, deleting the invalid certs, I don't know. Debug builds would  not work. So, I tried xTextHandler, another extension, and nothing works for it. So, why one works and the other, I cannot figure out.

NOTE: FBXCodeExtension does not show up in System Preferences -> Extensions. But it does show up  in the Xcode Editor menu.
